openbas-api/src/main/java/io/openbas/executors/caldera/service/CalderaExecutorService.java

```diff
@@ -87,8 +88,10 @@ public CalderaExecutorService(
             } else {
                 executorService.remove(config.getId());
             }
+            platformSettingsService.cleanMessage();
         } catch (Exception e) {
             log.log(Level.SEVERE, "Error creating caldera executor: " + e);
+            platformSettingsService.errorMessage("Executor Caldera is not responding, your exercises may be impacted.");
```

Comment:
So that means that if the service is not available at start, we'll always have the message until Caldera is up and the backend has been restarted ? Do you think we could have a process trying every minute for instance until it's up to avoid having to restart the backend ?

Reply:
Good point, it's changed in the last commit